### ISSUE TITLE
Hexiweaer as beacon

### ISSUE THREAD

**Issue by Nourfdss** [MikroElektronika/HEXIWEAR#17]
Hello everyone .
anyone tried to make the hexiwear act as beacon and send simple message.
thanks

**Comment by ChristopherRabotin** [comment]
Over Bluetooth? And with the default OS?

**Comment by Nourfdss** [comment]
Yes , no write a new code , I tried to edit the code but not working In the hexiwear
